backend/danswer/connectors/web/connector.py

```diff
@@ -70,21 +76,20 @@ def load_credentials(self, credentials: dict[str, Any]) -> dict[str, Any] | None
     def load_from_state(self) -> GenerateDocumentsOutput:
         """Traverses through all pages found on the website
         and converts them into documents"""
-        visited_links: set[str] = set()
+        visited_links: set[
+            str
+        ] = set()  # Holds the urls without trailing slash, assumes content is the same
         to_visit: list[str] = [self.base_url]
         doc_batch: list[Document] = []
 
-        # Edge case handling user provides HTML without terminating slash (prevents duplicate)
```

Comment:
better to let the redirect handle this, but visited_links will always contain the url without trailing slash